api/server/server.go

```diff
@@ -319,7 +330,7 @@ func (s *Server) startGears(ctx context.Context, cancel context.CancelFunc) {
 	fmt.Println(runHeader)
 	fmt.Printf("        v%s\n\n", version.Version)
 
-	logrus.Infof("Serving Functions API on address `%s`", listen)
+	logrus.WithField("type", s.nodeType).Infof("Serving Functions API on address `%s`", listen)
```

Comment:
Cosmetic thing:
```go
logrus.Infof("Serving Functions %v API on address `%v`", s.nodeType, listen)
```
this is what i've been thinking about.

Reply:
makes sense too. 1s

Reply:
+1 to @denismakogon suggestion, except drop API and add %s in place so it won't be `api API`. 

Orrrr, while we're nitting this, how about change it to this:

```go
logrus.WithField("type", s.nodeType).Infof("Fn serving on `%s`", listen)
```

Reply:
thank you sir may I please have another